test/PizzeriaTest.cpp

```diff
@@ -65,3 +64,28 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, completeOrderWithStubPizzaAndTwoMockPizzas) {
+    StrictMock<PizzaMock> strict;
+    NiceMock<PizzaMock> nice;
+    PizzaStub stub{"hawaii"};
+    EXPECT_CALL(strict, getName())
+        .WillOnce(Return("americana"));
+    EXPECT_CALL(strict, getPrice())
+        .WillOnce(Return(30));
+    EXPECT_CALL(strict, getBakingTime())
+        .WillOnce(Return(minutes(10)));
+    EXPECT_CALL(nice, getName())
+        .WillOnce(Return("inferno"));
+    EXPECT_CALL(nice, getPrice())
+        .WillOnce(Return(50));
+    EXPECT_CALL(nice, getBakingTime())
+        .WillOnce(Return(minutes(20)));
+    EXPECT_CALL(mock, sleep_for(_))
+        .Times(3);
```

Comment:
Nope. We don't want to ignore the argument.
_ means that we do not care what was passed to this function. We should expect that sleep_for was called with the baking time of each pizza.